<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitializationFileEditor</class>
 <widget class="QWidget" name="InitializationFileEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CutterRC Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="cutterRCLoaded">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDialogButtonBox" name="saveRC">
     <property name="maximumSize">
      <size>
       <width>649</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="ConfigFileEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="executeNow">
     <property name="standardButtons">
      <set>QDialogButtonBox::Retry</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
